src/library_wasmfs.js

```diff
@@ -349,8 +371,20 @@ FS.createPreloadedFile = FS_createPreloadedFile;
       __wasmfs_readdir_finish(state);
       return entries;
     }),
-    // TODO: mount
-    // TODO: unmount
+    mount: (type, opts, mountpoint) => {
+#if ASSERTIONS
+      if (typeof type == 'string') {
+        // The filesystem was not included, and instead we have an error
+        // message stored in the variable.
+        throw type;
```

Comment:
Where does the error message stored in `type` come from?

Reply:
I think `parseTools.js` (Specifically `makeRemovedFSAssert()`) will scan through and try to find any usages of globals like `NODEFS` that are used when `library_nodefs.js` isn't linked in and convert the variable value to a string.